services/pcs/src/pcs/cli.py

```diff
@@ -28,3 +28,9 @@
 def sync_run_rest_app():
     """Run the HTTP REST API."""
     asyncio.run(run_rest_app())
+
+
+@cli.command(name="publish-events")
+def sync_run_republish(all: bool = False):
+    """Republish all outbox events regardless of publish status."""
```

Comment:
the docstring should be the same as for publish_events in main.py

Reply:
Makes sense, made it so